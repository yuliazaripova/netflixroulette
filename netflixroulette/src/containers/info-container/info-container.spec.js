import * as React from 'react';

import InfoContainer from './info-container';

describe("InfoContainer component", () => {
  it("should render InfoContainer component", () => {
    const component = shallow(<InfoContainer/>);
    expect(component).toMatchSnapshot();
  })
})