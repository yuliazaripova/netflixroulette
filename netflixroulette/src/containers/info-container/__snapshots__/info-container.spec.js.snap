// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoContainer component should render InfoContainer component 1`] = `
<div
  className="info-container"
/>
`;
